<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updatePeopleWindow</class>
 <widget class="QDialog" name="updatePeopleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 74, 106);
</string>
  </property>
  <widget class="QLineEdit" name="updatePeoName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 97, 138);
border-radius:15px;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="updatePeoDescribe">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>401</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 97, 138);
border-radius:15px;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="updatePeoConnection">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 97, 138);
border-radius:15px;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 181, 205);
color: rgb(255, 255, 255);
border: 0px groove gary;
border-radius:5px;
selection-background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 181, 205);
color: rgb(255, 255, 255);
border: 0px groove gary;
border-radius:5px;
selection-background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="peopleNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="peopelDescribeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>介绍</string>
   </property>
  </widget>
  <widget class="QLabel" name="peopelNoteLable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>备注</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
